includes/admin/settings/display-settings.php

```diff
@@ -50,12 +50,17 @@ function edd_options_page() {
 		$key = key( $sections );
 	}
 
-	$registered_sections = edd_get_settings_tab_sections( $active_tab );
-	$section             = isset( $_GET['section'] ) && ! empty( $registered_sections ) && array_key_exists( $_GET['section'], $registered_sections ) ? sanitize_text_field( $_GET['section'] ) : $key;
+	$section = isset( $_GET['section'] ) && ! empty( $sections ) && array_key_exists( $_GET['section'], $sections ) ? sanitize_text_field( $_GET['section'] ) : $key;
```

Comment:
```suggestion
	$section = ! empty( $_GET['section'] ) && ! empty( $sections[ $_GET['section'] ] ) ? sanitize_text_field( $_GET['section'] ) : $key;
```

1. I prefer `! empty( $_GET['section'] )`  because we also don't want to end up with an empty one anyway.
2. Refactored the second conditional as that allows us to drop down from 3 checks to 2, which feels a bit simpler to me.